pkg/notes/mod.ts

```diff
@@ -182,6 +197,23 @@ noteHandlers.openapi(RenoteRoute, async (c) => {
   return c.json(res[1], 200);
 });
 
+noteHandlers[CreateReactionRoute.method](
+  CreateReactionRoute.path,
+  AuthMiddleware.handle({ forceAuthorized: true }),
+);
+noteHandlers.openapi(CreateReactionRoute, async (c) => {
+  const { id } = c.req.valid('param');
+  const req = c.req.valid('json');
+  const accountID = Option.unwrap(c.get('accountID'));
+
+  const res = await reactionController.create(id, accountID, req.emoji);
+  if (Result.isErr(res)) {
+    return c.json({ error: res[1].message }, 400);
+  }
+
+  return c.json(res[1], 200);
```

Comment:
```suggestion
    return c.json({ error: Result.unwrap(res).message }, 400);
  }

  return c.json(Result.unwrap(res), 200);
```
`Result` の内部に直接アクセスしないほうがよいです. `unwrap` で取り出すか `mapOrElse` などの変換処理を使ってください.